assets/src/components/mapPage/mapDisplay.tsx

```diff
@@ -55,18 +55,19 @@ const RouteVehicles = ({
   const vehicles = useVehiclesForRoute(socket, selectedVehicleRoute)
   return (
     <>
-      {filterVehicles(vehicles).map((vehicle: Vehicle) => {
-        const isSelected = vehicle.id === selectedVehicleId
-        return (
-          <VehicleMarker
-            key={vehicle.id}
-            vehicle={vehicle}
-            isPrimary={isSelected === true}
-            isSelected={isSelected}
-            onSelect={onPrimaryVehicleSelect}
-          />
-        )
-      })}
+      {filterVehicles(vehicles)
+        .filter((vehicle) => vehicle.id !== selectedVehicleId)
+        .map((vehicle: Vehicle) => {
+          return (
+            <VehicleMarker
+              key={vehicle.id}
+              vehicle={vehicle}
+              isPrimary={false}
+              isSelected={false}
+              onSelect={onPrimaryVehicleSelect}
```

Comment:
suggestion: Maybe for clarity this component could be renamed `SecondaryRouteVehicles`? And similarly `onPrimaryVehicleSelect` `onVehicleSelect`